packages/components/organisms/f-checkout/src/components/Checkout.vue

```diff
@@ -922,6 +922,15 @@ export default {
                     this.shouldShowSpinner = true;
                 }
             }, this.spinnerTimeout);
+        },
+
+        getReferralState () {
+            const cookieName = `je-rw-menu-referral-state-${this.restaurant.id}`;
+            const referralCookie = this.$cookies.get(cookieName);
+
+            return !referralCookie || !referralCookie.menuReferralState
+                ? 'None'
+                : 'ReferredByWeb';
```

Comment:
```suggestion
            return referralCookie && referralCookie.menuReferralState
                ? 'ReferredByWeb'
                : 'None';
```
as for me is more readable